src/mame/galaxian/galaxian.cpp

```diff
@@ -710,6 +710,52 @@ t) 'bagmanmc'
   - DSW bit 6 was previously used for "Bonus Lives" settings, but it has no effect
     in this set because of 'NOP' instructions from 0x3501 to 0x3507.
 
+u) uniwars
+
+    Uniwars Warp Analysis on an Irem PCB from PrSwan (Paul Swan)
+    ------------------------------------------------------------
+
+    - What is the warp effect?
+      The game PCB has several fly wires of which Red & Green relate to the warp effect.
+
+      Video PCB connections
+      Red   => IC K8 (7420)  pin 12
+      Green => IC F6 (74161) pin 14
+
+      The Green signal is a clock generated on the video PCB and the 161 is likely one of the video H/V counters.
+      It measured 10.44us.
+
+      The Red signal is input to the video PCB:
+      - Warp off => idle high
+      - Warp on  => Green clock
+
+      The effect can be manually triggered by disconnecting the plug and bridging the red & green wire.
+      The warp comes from the star field generator - with warp enabled all other graphics elements are unchanged.
+
+      There is a cut track isolating K8 pin 12 to allow the Red wire to control pin 12.
+
+      Trace of the K8 pin 12 cut track:
+      => IC K9 (7402)  pin  3 (input, 2-in NOR)
+      => IC E9 (7432)  pin 12 (input, 2 in OR)
+      => IC D9 (74173) pin  4 (Q output, latch)
+
+      Comparing with the Midway schematic suggests Irem IC K8 === Midway IC 3C and K8 pin 12 corresponds to ~256H* feeding Midway IC 3C pin 9.
+
+    - How is the effect triggered?
+      A spare IC position is populated with a 74LS157 mux connected as follows:
+
+      - pin 1 - S => IC C2 (LS259) pin 9
+      - pin 2 - A => not connected but evidence of prior solder, wire may have fallen off.
+      - pin 3 - B => Green wire
+      - pin 4 - Q => Red wire
+      - pin 15 - ~E => GND
```

Comment:
A floating input will behave like a logic high with poor noise immunity on a TTL chip due to how the common base amplifier input works.  It’s almost definitely supposed to be connected to something.